### PR TITLE
Update repository question

### LINKED ISSUE #26: Update Repository question
Feedback from the UU walk-in session:

- Delete the first sentence. Replace with “Where do you plan to make your software publicly available?”
- Remove warning
- Delete last sentence

### PR DESCRIPTION
**Pull request by sjvrijn**
Fixes #26 

- [x] Delete the first sentence. Replace with “Where do you plan to make your software publicly available?”
- [x] Remove warning
- [x] Delete last sentence
Assuming the 'last sentence' is the last one after removing the warning, i.e., the one on identifiers for releases

### CODE REVIEW

File: docassemble/SMPDecisionTree/data/questions/software.yml [hunk 1]
```diff
@@ -317,16 +317,12 @@ mandatory: public or reuse or multi_user
 question: |
   Repository
 subquestion: |
-  Should `${ software_name }` be publicly available? The most important consideration is that potential users of the software are able to get a copy they can use. This is often a public or self-hosted repository like GitHub or GitLab.
-
-  Please provide links to the software if this is already the case. This should be a repository with globally unique, persistent, and resolvable identifiers to each release.
-
-  > ⚠️ If your repository grows so large that you work predominantly with shallow clones, the repository might be the only complete version of your code. In this case, consider backing up your repository together with your data.
+  Where do you plan to make your software publicly available? The most important consideration is that potential users of the software are able to get a copy they can use. This is often a public or self-hosted repository like [GitHub](https://www.github.com) or [GitLab](https://www.gitlab.com).
```
Comment (OleMussmann) [inline docassemble/SMPDecisionTree/data/questions/software.yml:320]: Should we use "yours software" -> \`${ software_name }\` to be consistent with the other questions?

**Review by OleMussmann — CHANGES_REQUESTED** [review]
I don't want to "request changes", I'd like to "resolve a discussion". Ah well.

File: docassemble/SMPDecisionTree/data/questions/software.yml (hunk 1 above)
Comment (sjvrijn) [reply docassemble/SMPDecisionTree/data/questions/software.yml]: Good point, updated

**Review by OleMussmann — APPROVED** [review]
lgtm, thanks!

**Pull request merged**